src/_shared/scss/_adverts-capi.scss

```diff
@@ -30,6 +30,9 @@
 .adverts--paidfor {
     .adverts__header {
         position: relative;
+        -webkit-flex-direction: column;
```

Comment:
You only need to provide the `flex-direction` property here; the compilation from SCSS to vanilla CSS adds in all the engine-specific alternatives.

Reply:
done!